phoenix-core/src/it/java/org/apache/phoenix/end2end/ShowCreateTableIT.java

```diff
@@ -108,14 +104,42 @@ public void testShowCreateTableDefaultFamily() throws Exception {
         ResultSet rs = conn.createStatement().executeQuery("SHOW CREATE TABLE " + tableName );
         assertTrue(rs.next());
 
+        String expected = "CREATE TABLE " + tableName + "(K VARCHAR NOT NULL PRIMARY KEY, " +
+                "\"dF\".INT1 INTEGER, \"dF\".INT2 INTEGER, A.INT3 INTEGER, A.INT4 INTEGER, " +
+                "\"b\".INT5 INTEGER, B.INT6 INTEGER) " +
+                "IMMUTABLE_ROWS=true, DEFAULT_COLUMN_FAMILY='dF'";
+        assertTrue("Expected: :" + expected + "\nResult: " + rs.getString(1),
+                rs.getString(1).equals(expected));
+    }
+
+    @Test
+    public void testShowCreateTableDefaultFamilyDrop() throws Exception {
```

Comment:
something like testShowCreateTableDefaultFamilyNonConsecutive would be more descriptive.
The same goes for the similar tests